src/DoctrineServiceProvider.php

```diff
@@ -309,10 +310,14 @@ protected function isLumen()
     }
 
     /**
+     * Checks if configuration key "doctrine_presence_verifier" is defined and returns its value (true|false), if it
+     * doesn't returns true by default
+     *
      * @return bool
      */
-    protected function registerPresenceValidator()
+    protected function shouldRegisterDoctrinePresenceValidator()
     {
-        return (null !== @$this->app['config']['doctrine']['doctrine_presence_verifier']) ? $this->app['config']['doctrine']['doctrine_presence_verifier'] : true;
+        $configValue = @$this->app['config']['doctrine']['doctrine_presence_verifier'];
```

Comment:
`$this->app['config']->get('doctrine.doctrine_presence_verifier', true)` is more reliable